pandas/conftest.py

```diff
@@ -687,6 +689,24 @@ def float_frame():
     return DataFrame(tm.getSeriesData())
 
 
+# ----------------------------------------------------------------
+# Scalars
+# ----------------------------------------------------------------
+@pytest.fixture(
+    params=[
+        (Interval(0, 1), "interval[int64]"),
+        (Period("2019Q1", freq="Q"), "period[Q-DEC]"),
+        (
+            Timestamp("2011-01-01", tz="US/Eastern"),
+            DatetimeTZDtype(tz="US/Eastern"),
+        ),
+        (Timedelta(seconds=500), "timedelta64[ns]"),
```

Comment:
xref #38032 for a weird bug when the timedelta is in `ns`

Reply:
great thanks. testing catching edge cases, who would think!